opm/autodiff/GeoProps.hpp

```diff
@@ -150,7 +134,7 @@ namespace Opm
             // opmfil is hardcoded to be true. i.e the volume weighting is always used
             opmfil = true;
             if (opmfil) {
-                minPvFillProps_(grid, eclState, ntg);
+                minPvFillProps_(grid, eclState, pvol_, ntg);
```

Comment:
There is a slight logical flaw here: if this branch is not taken (although hardcoded now) pvol_ will not be initialized at all. Perhaps this is the time to remove the conditional.